doc/concepts/pipeline-concepts/pipeline/service.md

```diff
@@ -0,0 +1,41 @@
+# Service
+
+Service is a special type of pipeline that does not process data but provides
+a capability to expose it to the outside world. For example, you can use
+a service to expose a Jupyter™ notebook that has the most
+up-to-date version of your data.
+
+The following pipeline spec extract is an example of how you can expose your
+Jupyter notebook as a service by adding a `service` field:
+
+```json
+{
+    "input": {
+        "pfs": {
+            "glob": "/",
+            "repo": "input"
+        }
+    },
+    "service": {
+        "external_port": 30888,
+        "internal_port": 8888
+    },
+    "transform": {
+        "cmd": [
+            "start-notebook.sh"
```

Comment:
Is this the canonical way to launch a notebook? Check with Gabe or someone in eng?